content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-github-classroom.md

```diff
@@ -0,0 +1,41 @@
+---
+title: About GitHub Classroom
+intro: 'Teachers and school administrators can use {% data variables.product.prodname_classroom %} to create virtual classrooms, make and edit assignments, automatically grade assignment submissions, and more.'
+type: overview
+topics: 
+  - Fundamentals
+versions:
+  fpt: '*'
+---
+
+## About {% data variables.product.prodname_classroom %}
+
+{% data variables.product.prodname_classroom %} is a teaching tool that lets teachers and school administrators create and manage digital classrooms and assignments. You can create assignments for individual students or groups, set due dates, and track assignments on your teacher dashboard. Additionally, {% data variables.product.prodname_classroom %} has many features that simplify tasks like providing feedback, grading assignments, and integrating your existing teaching tools.
+
+## {% data variables.product.prodname_classroom %} features 
+
+{% data variables.product.prodname_classroom %} offers a variety of features to simplify teaching and learning.
+
+### Assignment templates
+
+With {% data variables.product.prodname_classroom %}, you can create assignments that use template repositories with boilerplate code, documentation, and other resources. Assignments with template repositories create assignment repositories with starter code for your students. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."
+
+### Autograding
+
+Through {% data variables.product.prodname_classroom %}, you can configure tests to automatically grade students' work each time they push to their assignment repository. To learn more about autograding with {% data variables.product.prodname_classroom %}, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/use-autograding)."
+
+### Connect a learning management system
```

Comment:
It may be a good idea to rationalize these headings so that they all use the name of the feature (a noun) or an action (with a verb at the start). 
```suggestion
### Ability to connect a learning management system
```